webpack/scenes/ContentViews/Details/Versions/ContentViewVersions.js

```diff
@@ -236,8 +265,10 @@ const ContentViewVersions = ({ cvId, details }) => {
                 </DropdownItem>]}
             />
           </Grid>
-        )}
-      displaySelectAllCheckbox={hasActionPermissions}
+          )}
+        </>
+        }
+      displaySelectAllCheckbox={hasActionPermissions && results?.length !== 0}
```

Comment:
This could use the same check for `renderActionButtons` as the Grid above right?

Reply:
Wait, I thought that results?.length !== 0 and !!results?.length are two different tests? :smiling_face_with_tear:

Reply:
They are a little different when it comes to null values due to the `!!` which tests "truthy"-ness vs a !== 0 which just compares with 0. So if results is null the results?.length will be null and null !==0 will be true..

Reply:
@sjha4  But would !!results?.length ensure that we have >0 results on our table? How about if I change the condition to results?.length > 0, would that work?

Reply:
@tazhibaevaaliya : That would work but using the same condition as above with `renderActionButtons` would make it more consistent in readability.